### DISCUSSION TITLE
Trackpad scroll?

### DISCUSSION THREAD

**Discussion by asobirov** [davidjerleke/embla-carousel#245] [Help] [answered]
Is it possible to scroll carousel using trackpad, for example using two fingers on laptop as in the example?
**Example:** [codesandbox](https://codesandbox.io/embed/nextjs-css-only-carousel-ipyij)

**Comment by davidjerleke** [comment, marked as answer]
Hi @asobirov,

I think you’re looking for the Wheel Gestures plugin.

- [Plugin docs](https://www.embla-carousel.com/plugins/wheel-gestures/)

Best,
David

**Reply by davidjerleke** [reply]
@asobirov did my answer help you?

**Reply by asobirov** [reply]
Hi, I have not yet tried it out. But, yeah, I think that is what I was looking for.

**Reply by davidjerleke** [reply]
I see. No stress, just curious. I would appreciate if you could mark my answer as correct if you find it to be helpful.

Cheers
David
